### PR TITLE
RSU Errors Page

### PR DESCRIPTION
**Pull request by mwodahl**
# PR Details
This PR Adds an RSU Errors page to the map view of the cv-manager.

## Description

This new RSU Errors page shows the online status and SCMS status of an RSU in table format and allows users to navigate to the RSU on the map. Additionally, users can generate a summary email for the RSU that includes the current errors and UTC timestamp.

## How Has This Been Tested?

This has been tested by running the cv-manager locally using Docker. 

## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [ X ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] My changes require new environment variables:
  - [ ] I have updated the docker-compose, K8s YAML, and all dependent deployment configuration files.
- [ ] My changes require updates to the documentation:
  - [ ] I have updated the documentation accordingly.
- [ X ] My changes require updates and/or additions to the unit tests:
  - [ X ] I have modified/added tests to cover my changes.
- [ X ] All existing tests pass.

### CODE REVIEW

File: webapp/src/features/menu/DisplayRsuErrors.tsx [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+            }
+            scms_status={getRSUSCMSDisplay(selectedRSU.properties.ipv4_address)}
+            hidden={emailHidden}
+            setHidden={setHidden}
+          />
+        </div>
+      ) : (
+        <div id="container" className="sideBarOn" style={{ width: '95%' }}>
+          <h1 className="h1" style={{ marginBottom: '1rem', width: '100%', textAlign: 'center' }}>
+            RSU Errors
```
Comment (drewjj) [inline webapp/src/features/menu/DisplayRsuErrors.tsx]: Is this table only displaying RSUs with errors? If not, maybe it makes more sense to call this RSU Status? I think the RSU navigation element is super useful for even working RSUs. You could look at changing the status text color for each RSU so it is visually easier to spot issues too.

**Review by drewjj — COMMENTED** [review]
Everything looks really good as a first swing at it! The email feature seems super useful but a little limited in the current implementation. Manually finding an RSU with an error and manually having the CV Manager send an email for that one RSU seems a little pointless to me. If there was a way to include all RSUs with issues or to generate a full report as PDF it might save the user time and be a little less tedious.

File: webapp/src/features/menu/DisplayRsuErrors.tsx (hunk 1 above)
Comment (mwodahl) [reply webapp/src/features/menu/DisplayRsuErrors.tsx]: Updated the table name to RSU Status and set text color to green/red depending on error status

**Comment by mwodahl** [comment]
Updated all of the display text on this new component to 'RSU Status' instead of 'RSU Errors' and added PDF generation for both the full RSU report and the RSU error report.

**Review by drewjj — APPROVED** [review]
This looks great and is going to be incredibly useful for the user experience. All tests pass too.

**Review by payneBrandon — CHANGES_REQUESTED** [review]
Just one question regarding formatting to remove the scrollbar 
image
Otherwise looks awesome!

**Comment by mwodahl** [comment]
Removed variable page length options from the RSU Status table so there isn't a horizontal scrollbar on the footer.

**Review by payneBrandon — APPROVED** [review]
lgtm!

**Review by payneBrandon — APPROVED** [review]
lgtm!

**Pull request merged**